docs/source/addons/theme.md

```diff
@@ -0,0 +1,230 @@
+
+/* Views */
+@ad          : 'pastanaga';
+@card        : 'pastanaga';
+@comment     : 'pastanaga';
+@feed        : 'pastanaga';
+@item        : 'pastanaga';
+@statistic   : 'pastanaga';
+
+/* Extras */
+@main        : 'pastanaga';
+@custom      : 'pastanaga';
+
+/*******************************
+            Folders
+*******************************/
+
+/* Path to theme packages */
+@themesFolder : '~volto-themes';
+
+/* Path to site override folder */
+@siteFolder  : "<name_of_your_theme>/theme";
+
+/*******************************
+         Import Theme
+*******************************/
+
+@import (multiple) "~semantic-ui-less/theme.less";
+@fontPath : "~volto-themes/@{theme}/assets/fonts";
+
+.loadAddonOverrides() {
+  @import (optional) "@{siteFolder}/@{addon}/@{addontype}s/@{addonelement}.overrides";
+}
+
+/* End Config */
+```
+
+3. Declare the theme as an add-on by adding its name to the value for the `addons` key in either `volto.config.js` or `package.json` of your project.
+4. After starting Volto, the theme should be active.
+   Now you can add overrides to the default theme in `src/theme`, same as you would in a project.
+5. Now you can safely delete your project's `theme` folder, since the one in the add-on will take precedence and a project can only have one active theme at a time.
+
+## Using your own theming escape hatch
+
+Volto theming uses SemanticUI theming capabilities to define and extend a theme for your site.
+However, while maintaining and playing well with the Semantic UI Volto base, using a traditional CSS approach can be done using the LESS preprocessor-based `extras` escape hatch.
+
+At the same time, one can either discard or complement the extras escape hatch and add your own, by customizing the `theme.js` module in Volto.
+
+```js
+import 'semantic-ui-less/semantic.less';
+import '@plone/volto/../theme/themes/pastanaga/extras/extras.less';
+
+// You can add more entry points for theming
+import '@kitconcept/volto-light-theme/theme/main.scss';
+```
+
+Customizing it is a special use case in Volto: add a `./@root/theme.js` file structure in your `customizations` folder in your add-on or project.
+
+You may want to do this to create a complete new theming experience adapted to your way of doing things that do not match the current Volto theming experience.
+For example, if you want to use another preprocessor in the theme, like SCSS.
+Maybe because your client forces you to have another entirely base of pre-made components based on another library other than Semantic UI:
+See {ref}`volto-custom-theming-strategy` for an example of a custom theme escape hatch.
+
+While building your own escape hatch for theming, you can use the preprocessor of your choice (in the example, SCSS) while maintaining the "base" Volto theme, but customizing it using the resultant CSS.
+
+You can see an example of such a theme in: https://github.com/kitconcept/volto-light-theme
+
+## Modify a custom theme from another add-on
```

Comment:
It's still not clear to me, what part of this section are instructions for the reusable theme, and which instructions belong to the add-on which slightly customizes the theme.
I stop bothering now, as I planned to take some days of from yesterday on. 
Thanks for the clarifications so far. I'll come back with questions later, like: "How would I use a reusable theme like kitconcept/volto-light-theme, but with another text color (probably $text-color ? But where? /theme/custom/_main.scss ?)"

Reply:
Yeah no worries, thanks for the valuable feedback! We will continue developing this and the story behind, and improving docs. The plan to work on this and the future of theming in Volto during Beethoven Sprint.